src/models/create_token_transaction.ts

```diff
@@ -242,6 +251,71 @@ class CreateTokenTransaction extends Transaction {
 
     return tx;
   }
+
+  /**
+   * Checks if this transaction is the creation of an NFT following the NFT Standard Creation.
+   * @see https://github.com/HathorNetwork/rfcs/blob/master/text/0032-nft-standard.md#transaction-standard
+   * @throws {NftValidationError} Will throw an error if the NFT is not valid
+   *
+   * @param {Network} network Network to get output addresses first byte
+   * @returns {void} If this function does not throw, the NFT is valid
+   */
+  validateNftCreation(network: Network): void {
+    // An invalid transaction will fail here too
+    this.validate();
+
+    // No need to check the tx version, it is enforced by the class constructor
+
+    /*
+     * NFT creation must have at least a DataScript output (the first one) and a Token P2PKH output.
+     * Also validating maximum outputs of transactions in general
+     */
+    if (this.outputs.length < 2) {
+      throw new NftValidationError(`Tx has less than the minimum required amount of outputs`);
+    }
+
+    // Validating the first output
+    const firstOutput = this.outputs[0];
+
+    // NFT creation DataScript output must have value 1 and must be of HTR
+    if (firstOutput.value !== 1 || !firstOutput.isTokenHTR()) {
+      throw new NftValidationError(`First output is not a valid NFT fee`);
+    }
+    // NFT creation Datascript must be of type data
+    if (!(firstOutput.parseScript(network) instanceof ScriptData)) {
+      throw new NftValidationError(`First output is not a fee DataScript`);
```

Comment:
I would remove the word `fee` in this error message.

Reply:
Adjusted on [ee33a8f](https://github.com/HathorNetwork/hathor-wallet-lib/pull/378/commits/ee33a8f3995b1bb84f56d6459daefbec21a89e1e).